bin/cqlsh.py

```diff
@@ -2373,9 +2408,11 @@ def main(options, hostname, port):
                       tty=options.tty,
                       completekey=options.completekey,
                       browser=options.browser,
+                      debug=options.debug,
```

Comment:
This change was not necessary, but it was intermittent yet, I think valid to set `debug` flag early so that it is possible to use it to dump some info even if the connection is not established.